accounts/abi/bind/backends/simulated.go

```diff
@@ -210,6 +212,13 @@ func (b *SimulatedBackend) NonceAt(ctx context.Context, contract common.Address,
 	return stateDB.GetNonce(contract), nil
 }
 
+func (b *SimulatedBackend) SetMarketGasPrice(gasPrice int64) {
```

Comment:
let's add a go style comment here describing this function and a potential use case

Reply:
done!